README.md

```diff
@@ -1,10 +1,10 @@
 # [![Fluture](logo.png)](#butterfly)
 
-[![Chat](https://badges.gitter.im/fluture-js/Fluture.svg)](https://gitter.im/fluture-js/Fluture)
-[![NPM Version](https://badge.fury.io/js/fluture.svg)](https://www.npmjs.com/package/fluture)
-[![Dependencies](https://david-dm.org/fluture-js/Fluture.svg)](https://david-dm.org/fluture-js/Fluture)
-[![Build Status](https://travis-ci.org/fluture-js/Fluture.svg?branch=master)](https://travis-ci.org/fluture-js/Fluture)
-[![Code Coverage](https://codecov.io/gh/fluture-js/Fluture/branch/master/graph/badge.svg)](https://codecov.io/gh/fluture-js/Fluture)
+[![Build Status](https://img.shields.io/travis/fluture-js/Fluture/master.svg)](https://travis-ci.org/fluture-js/Fluture)
+[![Code Coverage](https://img.shields.io/codecov/c/github/fluture-js/Fluture/master.svg)](https://codecov.io/gh/fluture-js/Fluture/branch/master)
+[![NPM Downloads](https://img.shields.io/npm/dm/fluture.svg)](https://www.npmjs.com/package/fluture)
+[![Gitter Chat](https://img.shields.io/gitter/room/fluture-js/Fluture.svg)](https://gitter.im/fluture-js/Fluture)
+[![StackExchange](https://img.shields.io/stackexchange/stackoverflow/t/fluture.svg)](https://stackoverflow.com/questions/tagged/fluture)
```

Comment:
This image is not loading for me.

Reply:
Yeah, it appears that the stackoverflow api might be rate liited. The image only loads very rarely for me, and on https://shields.io/ the badge shows as "vendor unesponsive":

![latest-screenshot](https://user-images.githubusercontent.com/1217745/43207438-679d0e36-9028-11e8-859e-ede287fa8b3a.png)

I have seen it working a few times though, by zealously refreshing the page. It might not be worth including in the docs if it rarely loads, though. However, when the image doesn't load, the `alt` is rendered ("StackExchange"). This text still has some value, and links to the appropriate section on StackExchange.

Reply:
I don't think we should include an unreliable badge. We could provide a text link instead.

> However, when the image doesn't load, the `alt` is rendered ("StackExchange").

I don't see the alt text in Safari on macOS.

Reply:
Safari is the new Internet Explorer.

Anyway, what do you think of rendering something like this instead?

Community resources | Badges
--------------------|------
[Build Status][]    | ![Build Status Badge][]
[Code Coverage][]   | ![Code Coverage Badge][]
[Dependencies][]    | ![Greenkeeper Badge][] ![David DM Badge][]
[NPM Package][]     | ![NPM Version Badge][] ![NPM Downloads Badge][]
[Gitter Chat][]     | ![Gitter Chat Badge][]
[StackOverflow][]   | ![StackOverflow Questions Badge][]

[Build Status]: https://travis-ci.org/fluture-js/Fluture
[Code Coverage]: https://codecov.io/gh/fluture-js/Fluture/branch/master
[Dependencies]: https://david-dm.org/fluture-js/Fluture
[NPM Package]: https://www.npmjs.com/package/fluture
[Gitter Chat]: https://gitter.im/fluture-js/Fluture
[StackOverflow]: https://stackoverflow.com/questions/tagged/fluture
[Build Status Badge]: https://img.shields.io/travis/fluture-js/Fluture/master.svg
[Code Coverage Badge]: https://img.shields.io/codecov/c/github/fluture-js/Fluture/master.svg
[Greenkeeper Badge]: https://badges.greenkeeper.io/fluture-js/fluture-retry.svg
[David DM Badge]: https://img.shields.io/david/fluture-js/Fluture.svg
[NPM Version Badge]: https://img.shields.io/npm/v/fluture.svg
[NPM Downloads Badge]: https://img.shields.io/npm/dm/fluture.svg
[Gitter Chat Badge]: https://img.shields.io/gitter/room/fluture-js/Fluture.svg
[StackOverflow Questions Badge]: https://img.shields.io/stackexchange/stackoverflow/t/fluture.svg

Reply:
Everything just loaded, so I took a quick screenshot before it was gone! 'Gave me Pokemon Snap flashbacks! That's what it looks like:

![latest-screenshot](https://user-images.githubusercontent.com/1217745/43211731-851d1532-9032-11e8-9e7f-965e3f4678f3.png)

Reply:
The table looks fine, but I don't think it belongs at the beginning of the document. I lean towards sticking with a single row of badges.

Reply:
Some of these badges are not informative, and are supposed to be used as links. That's where this approach collapses. I'm not sure how to proceed.

Reply:
> I lean towards sticking with a single row of badges.

Agreed. Alright. I will just pretend that all browsers show the alt text, and treat the badges like menu items. Who uses Safari anyway? ;)